### PR TITLE
Feature/copy button

### PR DESCRIPTION
**Pull request by Katarzyna-Kadziolka**

### CODE REVIEW

File: doskvol-tools/src/components/common/BaseButton.vue [hunk 1]
```diff
@@ -1,31 +1,38 @@
 <script setup lang="ts">
-defineProps<{
-  name: string
-}>();
+
+withDefaults(defineProps<{
+  name: string,
+  isPrimary: boolean
```
Comment (Morasiu) [inline doskvol-tools/src/components/common/BaseButton.vue:5]: Should be default to false

File: doskvol-tools/src/components/generator/BaseGenerator.vue [hunk 2]
```diff
@@ -24,7 +25,10 @@ defineEmits<{
         <span class="base-generator_title">{{title}}</span>
         <slot name="results"></slot>
       </div>
-      <BaseButton :name="t('generator-ui.generate')" @click="$emit('generateRandom')"/>
+      <div class="base-generator_buttons-container">
+        <BaseButton :name="t('generator-ui.generate')" @click="$emit('generateRandom')" :isPrimary="true"/>
```
Comment (Morasiu) [inline doskvol-tools/src/components/generator/BaseGenerator.vue:29]: Will this work?
```vue
<BaseButton :name="t('generator-ui.generate')" @click="$emit('generateRandom')" isPrimary />
```

**Review by Morasiu — CHANGES_REQUESTED** [review]

File: doskvol-tools/src/components/common/BaseButton.vue (hunk 1 above)
Comment (Katarzyna-Kadziolka) [reply doskvol-tools/src/components/common/BaseButton.vue:5]: Ok, I changed

File: doskvol-tools/src/components/generator/BaseGenerator.vue (hunk 2 above)
Comment (Katarzyna-Kadziolka) [reply doskvol-tools/src/components/generator/BaseGenerator.vue]: It works, so I changed it

**Pull request merged**